newapi/ooniapi/probe_services.py

```diff
@@ -171,12 +171,20 @@ def check_in():
         # TODO: add fallback
         test_items = []
 
+    try:
+        torconf = _load_json(current_app.config["TOR_TARGETS_CONFFILE"])
+        psconf = _load_json(current_app.config["PSIPHON_CONFFILE"])
+        conf = dict(tor=torconf, psiphon=psconf)
+    except Exception as e:
+        log.error(str(e), exc_info=1)
+        conf = {}
+
     resp = dict(
         v=1,
         tests={
             "web_connectivity": {"urls": test_items},
         },
-        conf={},
+        conf=conf,
```

Comment:
Shouldn't this go nested inside of the `tests` key under the testname it's relevant to?

Reply:
`tests` contains data about the tests to be run right now and should not be cached on the probes

`conf` contains more persistent configuration and should be cached by the probes